src/main/java/TaskList.java

```diff
@@ -58,17 +62,24 @@ public void addEvent(String taskKey) {
             System.out.println(err.getMessage());
             return;
         }
-        Task newTask = new EventTask(taskTitle, deadline);
-        this.itemList.add(newTask);
-        System.out.println(
-                "----------------------------" +
-                        "----------------------------\n" +
-                        "Got it. I've added this task:\n"
-                        + "  " + newTask + "\n"
-                        + "Now you have " + this.itemList.size() + " tasks in the list."
-                        + "\n"
-                        + "--------------------------------------------------------"
-        );
+
+        try {
+            DateTimeFormatter formatter = DateTimeFormatter.ofPattern("d/MM/yyyy HHmm");
+            System.out.println(deadline);
+            Task newTask = new EventTask(taskTitle, LocalDateTime.parse(deadline, formatter));
+            this.itemList.add(newTask);
+            System.out.println(
```

Comment:
For this portion, there might be an issue if, for any reason, the addition of the task fails and an error is not thrown. This will cause the system to print out the success statement despite failing to add the task. May I suggest that a possible structure would be to return a boolean value for a success addition, and nest the success message within the if-statement of the addTask.